Src/AutoMoq/AutoMockPropertiesCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// A command that populates all public writable properties/fields of a mock object with anonymous values.
+    /// </summary>
+    public class AutoMockPropertiesCommand : ISpecimenCommand
```

Comment:
@moodmosaic were you perhaps referring to my last comment where I pointed out the difference between `fixture.Create<IHasProperties>()` and `fixture.Create<Mock<IHasProperties>>().Object`? If so, yes, it would be a POLA violation.